src/app.exec.ts

```diff
@@ -166,6 +180,43 @@ async function exec(ass: RV1, pkgs: PackageSpecification[], opts: {env: boolean}
 
 ////
 
+import {readLines} from "deno/io/buffer.ts"
+
+async function extract_shebang(path: Path) {
+  const f = await Deno.open(path.string, { read: true })
+  const line = (await readLines(f).next()).value
+  const shebang = line.match(/^\s*#!(\S+)$/)?.[1]
```

Comment:
This might be overly accepting. I bet a lot of tools don't accept leading spaces. but that shouldn't break anything.

This looks like it won't handle /usr/bin/env ruby, for example. That's probably ok, but just noting it.

Reply:
I added `env` support.

I am accepting leading spaces because (amazingly) I discovered most interpreters also do this.